server/src/main/java/org/elasticsearch/index/mapper/AbstractGeometryFieldMapper.java

```diff
@@ -76,18 +82,19 @@
         Parsed parse(XContentParser parser, AbstractGeometryFieldMapper mapper) throws IOException, ParseException;
     }
 
-    public abstract static class Builder<T extends Builder, FT extends AbstractGeometryFieldType>
+    public abstract static class Builder<T extends Builder<T, FT>, FT extends AbstractGeometryFieldType>
             extends FieldMapper.Builder<T> {
         protected Boolean ignoreMalformed;
         protected Boolean ignoreZValue;
+        protected boolean indexed = true;
```

Comment:
Is it possible that this pull request causes https://github.com/elastic/elasticsearch/issues/69401? In 7.9.0 geo_point fields are always indexed, that is a regression, since it was possible to make geo_point fields not indexable in earlier versions.